src/design/layers.css

```
@@ -18,64 +19,73 @@
+  .score-event-buttons { display: grid; grid-template-columns: repeat(4, minmax(0, 1fr)); gap: 0.5rem; }
+  .score-event-buttons button { min-height: 3rem; }
+  .score-event-buttons .score-event-button--no-ball { border-color: #ef233c; background: #ef233c; color: white; }
+  .score-event-buttons .score-event-button--neutral { border-color: #cbd5e1; background: white; color: #0f172a; }
 
   .dark .score-page {
     --score-panel: #1d293d;
     --score-panel-soft: #111a2a;
     --score-muted: #91a1bb;
     --score-green: #08bf8a;
+    --score-red: #ff3b30;
     color: #f8fafc;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)